### DISCUSSION TITLE
Is there a recommendation for what to do about reorganising repos?

### DISCUSSION THREAD

**Discussion by danielrosehill** [alefragnani/vscode-project-manager#767] [Q&A] [answered]
Hi!
Thanks for making this. Truly a huge time saver.
I have one question
My /git folder has spiralled a little out of control lately with a lot of repos
I'm trying to do a little bit of housekeeping ... batching repos related to the same project under parent folders etc
I know that doing so is going to break the paths in the JSON
Any chance there's a feature designed to handle this / symlink creation etc?
Any recommendations appreciated!

**Comment by alefragnani** [comment, marked as answer]
Hi @danielrosehill ,

I suppose this is a duplicate of #768 , so let’s centralize the conversation there.

Hope this helps
